README.md

```diff
@@ -0,0 +1,13 @@
+# Survey Form
+
+#### Contributors
+
+Charles Byakutaga
+
+- charlesbyaxy@gmail.com
+- [GitHub](https://github.com/Byaxy)
+
+Brighton Mtandi
+
+- mtandibrighton0102@gmail.com
+- [GitHub](https://github.com/brighton0102)
```

Comment:
Kindly adhere to the [Microverse README template](https://github.com/microverseinc/readme-template) structure, because the template provided by Microverse is well documented and makes your repository look more professional.

Reply:
- kindly, use a couple of emojis and/or gifs in the review. They make the code review more human-friendly :alien::robot:.